GLib/2.75.4/Recipe

```diff
@@ -0,0 +1,21 @@
+# Recipe for version 2.75.4 by Hisham Muhammad <hisham@gobolinux.org>, on Mon Mar 6 18:59:43 -03 2023
+# Recipe (MakeRecipe) for GLib by  Ozgur Ulukir, on Mon Dec 20 01:48:37 EET 2004
+compile_version=2.0
+url="https://gitlab.gnome.org/GNOME/glib/-/archive/2.75.4/glib-2.75.4.tar.bz2"
+file_size=6502073
+file_md5=4d4f5f997ea9a2da7585f88acacf0310
+recipe_type=meson
+
+meson_variables=(
+    -Dselinux=disabled
+)
+
+pre_build() {
+    rmdir ../subprojects/gvdb
```

Comment:
Hey @hishamhm !
Any chance you remember the reason for adding this here? This line was not present in previous Glib recipes and causes an error: `rmdir: failed to remove '../subprojects/gvdb': Directory not empty`for me (and another user in IRC chat).

I'm inclined to change it to an `rm -rf`, but not sure why it is needed. The compilation succeed with and without this particular pre-build operation.